first init

blackrainbowtest opened the pull request:
If you find errors in the code, please leave a comment on the error line.

Thank you for your work

blackrainbowtest committed:
first init

LusineAvetisyan01 commented on index.js:
```diff
@@ -0,0 +1,35 @@
+    if(userText.trim()) {
+        const lettersCount = userText.replace(/[^A-Za-zА-Яа-я]/gu, '').length;
+        const wordsCount = userText.split(/\s+/).filter((word) => word.length > 0).length;
+        const sentencesCount = userText.split(/[.!?]+/).filter((sentence) => sentence.length > 0).length;
```
The sentence logic is not quiet correct։ in some cases the results gives you one redundant sentence more than it should be. For example if you type "2.5" the sentences count is 2 which is incorrect.

blackrainbowtest replied:
Thank you very much for your feedback.

 Thanks to your help, I fixed that error.

LusineAvetisyan01 commented on style.css:
Styling is overall good but you could add hover styling to all buttons, currently only the ‘count’ button has that.

LusineAvetisyan01 reviewed (commented):
Reset button is a cool feature, but it’s not clear at first if it's related to the count reset or the background reset so you could somehow fix that.
Picked names are on point. 
Overall functionality and complexity are good too!

GarikSaroyan commented:
1. the buttons are good, but first, I think if I press the reset button, the text will be deleted, but it won't happen.
2.  rndColor and resetColor were not used in index.js
3. you write in css all names by id but it is not recommended 
4. in the index.js in 16,17,18 lines you can write for example                 letters.innerHTML = '0' and ...

blackrainbowtest commented:
> 1. the buttons are good, but first, I think if I press the reset button, the text will be deleted, but it won't happen.
> 2. rndColor and resetColor were not used in index.js
> 3. you write in css all names by id but it is not recommended
> 4. in the index.js in 16,17,18 lines you can write for example                 letters.innerHTML = '0' and ...

1. The reset button is designed to reset the background color. I changed the name "reset BG" to make it clearer.
2. This will be removed in the next update.
3. Thanks for the advice, however, since there were only 4 elements that needed to be styled, and they already had an id, I decided not to use classes.
4. Unfortunately, letters and other variables are declared in the If block and they are not visible in the Else block, but in the next update these variables are also removed and the process is simplified.

Thanks for your feedback.
